README.md

```diff
@@ -53,43 +54,41 @@ fud exec depth.futil --to interpreter-out -s verilog.data depth.data > depth.txt
 python3 parse_data.py -di temp.txt
 ```
 
+What just happened? Below we walk through the six commands we issued above, pointing out the other options that we could have used.
+
 First, `make fetch` downloads some [GFA][] data files into the `./test` directory.
 
-To build the odgi graph files from the GFA files, run `make test/*.og`.
+Second, `make test/*.og` builds the odgi graph files from those GFA files.
 
-Then, `python3 calyx_depth.py -o depth.futil` generates the hardware accelerator and writes it to a file named `depth.futil`. The commands to generate a node depth hardware accelerator in calyx include:
+Third, we generate the hardware accelerator and write it to a file named `depth.futil`. The commands to generate a node depth hardware accelerator in calyx include:
 
 1. `python3 calyx_depth.py -o depth.futil`
-2. `python3 calyx_depth.py -a <filename> -o depth.futil`
-3. `python3 calxy_depth.py -n=MAX_NODES -e=MAX_STEPS -p=MAX_PATHS -o depth.futil`
+2. `python3 calxy_depth.py -n=MAX_NODES -e=MAX_STEPS -p=MAX_PATHS -o depth.futil`
+3. `python3 calyx_depth.py -a <filename.og> -o depth.futil`
```

Comment:
Just an experiment, adding the expected extensions of the various files. If you like it, we can do it all over. If you hate it, I believe this is the only spot I added it!

Reply:
That's probably a good idea to add the expected file extensions. I can go through and change it if you don't get around to it first

Reply:
Just curious, what was the reason for swapping the order of 2 and 3?

Reply:
That's probably a good idea to add the expected file extensions. I can go through and change it if you don't get around to it first

Reply:
Heh, no hard reason! I guess in my head it flows better this way, since I view the supplying of `filename.og` to be a more heavyweight way to do it. Happy to swap it back!

Reply:
That is fair; it definitely takes more time for the code to infer the parameters. In my head I was viewing it as more of a burden on the user to have to specify each parameter

Reply:
File extensions updated!

Reply:
Heh so maybe I would not have screwed them up after all...